improver_tests/acceptance/SHA256SUMS

```diff
@@ -460,14 +460,20 @@ df2a6ed228f820901aa00ff0ac2b6e68ae14f0751b2d887fbde7a657845b2428  ./nowcast-opti
 35287094ae4710738ebaf7fe417ee5d9f6b489e4e9d1c895be6e08b31e54b775  ./orographic_enhancement/basic/wind_direction.nc
 9bf398af629f0a4b7b51158fe40ed2581d271801ae1d925b80df84c208d79120  ./orographic_enhancement/basic/wind_speed.nc
 636fb4f195bb16fd0eff704660618ce1e203fb712905d7740a058ff589677364  ./orographic_enhancement/boundary_height/kgo_hi_res.nc
-8e73bf41e6d275f80ef27cb19491f70b55cd72551f504f4ba9b8a86bc619bd6f  ./phase-change-level/basic/hail_rain_kgo.nc
-16c0578e606553d4e041cf75e3bfbdbfdb78a0dc02f1f23099b580307951a9d0  ./phase-change-level/basic/land_mask.nc
-9623c8f2096b440b69b7e0d448a457ec70b10f88df6669cd78572642673e74a7  ./phase-change-level/basic/orog.nc
-7beb5a535688252eb81222d2bfa13a82306d4c3da3990b1ed09f850431495fbb  ./phase-change-level/basic/sleet_rain_kgo.nc
-a87b4eb0ae9dfac3cdffb75dba94750e047fe618b4bb469f5e2262819bfd2318  ./phase-change-level/basic/sleet_rain_unfilled_kgo.nc
-ce8c714663974d489b25516f752c21424c52182dacc930f78167c6fbfe7e3ac2  ./phase-change-level/basic/snow_sleet_kgo.nc
-8693b7061a6a3048b193b084e23f9c782a655426764a2c3d1b77ea456d883c09  ./phase-change-level/basic/wbti.nc
-95c8e9238f70628bfa19276cf32c401460fee518411a51286c25de3598656e85  ./phase-change-level/basic/wet_bulb_temperature.nc
+16c0578e606553d4e041cf75e3bfbdbfdb78a0dc02f1f23099b580307951a9d0  ./phase-change-level/land_mask.nc
+9623c8f2096b440b69b7e0d448a457ec70b10f88df6669cd78572642673e74a7  ./phase-change-level/orog.nc
+3d8a9cf5d63c2405654ee84d6b9765e5f10f1dd79108d6e5c8a26a30bf0ff922  ./phase-change-level/with_id_attr/hail_rain_kgo.nc
+3a551098e4413b9f1e531064e2bfbebc44dd2031164ecf40257255d968d25e6b  ./phase-change-level/with_id_attr/sleet_rain_kgo.nc
+9eeb81d76ca5dee359a47c181a296e0dee110b69dc223b491990efb90d2f0236  ./phase-change-level/with_id_attr/sleet_rain_unfilled_kgo.nc
+6b7b0ea690520e1f7ebfd7c8e7cc7c645964010f98cbf49d472d49fdb4ef3a4b  ./phase-change-level/with_id_attr/snow_sleet_kgo.nc
+651c80cd4ab7c720c59c6c0c2c4a2ade85aee81c2ab0c697b577ca87a36d57f2  ./phase-change-level/with_id_attr/wbti.nc
+c2b71855f260544ca75aaa5900f3297b58cee1b3b64eec5805662e1927d5fdab  ./phase-change-level/with_id_attr/wet_bulb_temperature.nc
+8e73bf41e6d275f80ef27cb19491f70b55cd72551f504f4ba9b8a86bc619bd6f  ./phase-change-level/without_id_attr/hail_rain_kgo.nc
+7beb5a535688252eb81222d2bfa13a82306d4c3da3990b1ed09f850431495fbb  ./phase-change-level/without_id_attr/sleet_rain_kgo.nc
+a87b4eb0ae9dfac3cdffb75dba94750e047fe618b4bb469f5e2262819bfd2318  ./phase-change-level/without_id_attr/sleet_rain_unfilled_kgo.nc
+ce8c714663974d489b25516f752c21424c52182dacc930f78167c6fbfe7e3ac2  ./phase-change-level/without_id_attr/snow_sleet_kgo.nc
+8693b7061a6a3048b193b084e23f9c782a655426764a2c3d1b77ea456d883c09  ./phase-change-level/without_id_attr/wbti.nc
+95c8e9238f70628bfa19276cf32c401460fee518411a51286c25de3598656e85  ./phase-change-level/without_id_attr/wet_bulb_temperature.nc
 d6f4cc94f3e317b9cad58f5f2a7f4147532b21d9bddd89df79158b44a18ee95e  ./phase-mask/rain/kgo.nc
```

Comment:
We could save space by just having one version of the inputs (`with_id_attr`) as the id attribute will be ignored by the `without` test.

Reply:
I've reduced the duplication in the test data as suggested.